features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesPresenter.kt

```diff
@@ -183,6 +182,18 @@ class MessagesPresenter @AssistedInject constructor(
             enableVoiceMessages = featureFlagsService.isFeatureEnabled(FeatureFlags.VoiceMessages)
         }
 
+        var dmUserVerificationState by remember { mutableStateOf<IdentityState?>(null) }
+
+        LifecycleResumeEffect(Unit) {
+            if (room.isDm && room.isEncrypted) {
+                localCoroutineScope.launch {
+                    val dmUserId = room.getMembers().getOrNull()?.find { it.userId != room.sessionId }?.userId
```

Comment:
I am wondering if getMembers will always returns the whole member list. Maybe for DM this is OK.

Else you may want to use
```kotlin
val membersState by room.membersStateFlow.collectAsState()
val otherMember = room.getDirectRoomMember(membersState)
```
instead?

Reply:
I think you're right, I just noticed the warning in its docs:

> Get the members of the room. Note: generally this should not be used, please use [membersStateFlow] and [updateMembers] instead.